app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -435,6 +459,24 @@ class HomeActivityTest {
     }
   }
 
+  /**
+   * # Dependency graph:
+   *
+   *      Fractions
+   *         |
+   *        |
+   *       v
+   * Test topic 1                     Ratios
+   *    \                              /
+   *     \                           /
+   *       -----> Test topic 2 <----
```

Comment:
```suggestion
   *       -----> Test topic 1 <----
```

This & above aligns better with test names.